### ISSUE TITLE
[BUG] CLI clear_offline_gateways doesn't work

### ISSUE THREAD

**Issue by vgeneves** [wirepas/backend-client#275]
The clear_offline_gateways doesn't work. Each time the command is launched the following error appears :
10:12.32 | wm-gw-cli > clear_offline_gateways
Something went wrong:Protocol message GenericMessage has no "gw_id" field.

**Software versions:**
 - OS: WSL2
 - Python version: 3.7
 - backend-client : 1.3.2

**Comment by ErkkiSalonenW** [comment]
When I type "clear_offline_gateways" and press enter, empty line is printed and returned back to CLI. What commands have to typed before?

**Comment by vgeneves** [comment]
I don't type any commands before.

**Comment by PFigs** [comment]
Are your connection settings correct?

**Comment by vgeneves** [comment]
Yes I am. I am able to launch the CLI without problem. And most of the other commands are working fine.

**Comment by PFigs** [comment]
The cli often starts ok even when the connection setttings are incorrect, eg, secure vs insecure. 

Does not seem to be the case, though. The error seems to indicate that the instantiation of the message object is not working as expected. The gw_id is inherited from the wirepas-messaging (see https://github.com/wirepas/backend-client/blob/master/wirepas_backend_client/messages/decoders/generic.py)
